components/StyledInputAmount.js

```diff
@@ -83,7 +83,10 @@ const StyledInputAmount = ({
       inputMode="decimal"
       defaultValue={isUndefined(defaultValue) ? undefined : defaultValue / 100}
       value={isControlled ? getValue(value, rawValue) : undefined}
-      onChange={e => dispatchValue(e, parseValueFromEvent(e, precision))}
+      onChange={e => {
+        e.stopPropagation();
```

Comment:
I had the `StyledInputAmount` inside a `StyledRadioList` and selecting something for the input amount triggered a change in the radio list input. Because the input styles don't match it was crashing the page.